tests/BuildScriptGenerator.Tests/Node/NodeBuildScriptGenerationTest.cs

```diff
@@ -331,7 +366,7 @@ public void GeneratedScript_ZipsNodeModules_IfZipNodeProperty_IsTarGz()
             repo.AddFile(PackageJsonWithBuildScript, NodeConstants.PackageJsonFileName);
             var context = CreateScriptGeneratorContext(repo);
             context.LanguageVersion = "8.2.1";
-            context.Properties["compress_node_modules"] = "tar-gz";
+            context.Properties[NodeConstants.CompressNodeModulesPropertyKey] = NodeConstants.TarGzNodeModulesOption;
```

Comment:
here.. I was thinking if its better way to just know how you zipped rather than doing what you are doing here, current implementation works no doubt, my point is if my proposal is cleaner way to do it